p2p/src/authenticated/types.rs

```diff
@@ -1,6 +1,6 @@
 use bytes::{Buf, BufMut, Bytes};
 use commonware_codec::{
-    varint, Encode, EncodeSize, Error, RangeConfig, Read, ReadExt, ReadRangeExt, Write,
+    varint::VarUInt, Encode, EncodeSize, Error, RangeConfig, Read, ReadExt, ReadRangeExt, Write,
```

Comment:
To avoid stutter here, I recommend just calling the object in `varint` both `SInt` and `Uint`